### PR TITLE
Add tony-lein chaska file

### LINKED ISSUE #27: tony-lein Workflow for adding Chaska to the Repository
## Chaska, MN
- [x] Create an issue
- [x] Create a branch
- [x] Make changes
- [ ] Collaborate / Discussion
- [ ] Get approval to ship :+1: 
- [ ] Merge

### PR DESCRIPTION
**Pull request by tlcoder**
closes #27

### CODE REVIEW

File: tony-lein-chaska.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+## Chaska, MN
+
+### Restaurants
+- McDonalds
+
+### Places to Go
```
Comment (tlcoder) [inline tony-lein-chaska.md:7]: Wow :joy_cat:

**Review by tlcoder — COMMENTED** [review]
Amazing

File: tony-lein-chaska.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## Chaska, MN
+
+### Restaurants
+- McDonalds
```
Comment (skizdigidy) [inline tony-lein-chaska.md:4]: Yummy cheeseburgers 😋

Comment (tlcoder) [reply tony-lein-chaska.md:4]: :hamburger:

**Review by skizdigidy — APPROVED** [review]
I approve merging these changes

**Pull request merged**